tests/test_stream.py

```diff
@@ -0,0 +1,41 @@
+from lomond.stream import WebsocketStream, FrameParser
+from lomond.errors import ProtocolError
+import pytest
+
+
+@pytest.fixture
+def stream():
+    return WebsocketStream()
+
+
+def test_constructor(stream):
+    assert isinstance(stream, WebsocketStream)
```

Comment:
It's mostly setting private variables in the constructor, and no parameters. Not sure what test could be written that would catch anything that the `feed` tests wouldn't catch.